automatic/selenium-chromium-edge-driver/update.ps1

```diff
@@ -36,14 +34,14 @@ function global:au_SearchReplace {
 }
 
 function global:au_GetLatest {
-  # We use the available package on CCR as the source of truth
-  $packageVersion = Get-EdgePackageVersion -PackageName 'microsoft-edge'
-  $escapedVersion = [regex]::Escape($packageVersion)
+  $edgeDriverStableVersionURL = "https://msedgewebdriverstorage.blob.core.windows.net/edgewebdriver/LATEST_STABLE"
```

Comment:
Please move this up to where `$releases` is on line 3 and call it that. These files have a format that we should follow.

Reply:
Thx, makes sense. Didn't know the convention as this is my first PR here